src/Application/Actions/Donations/Create.php

```diff
@@ -91,6 +91,13 @@ protected function action(): Response
                     // See https://stripe.com/docs/api/payment_intents/object
                     'amount' => (100 * $donation->getAmount()),
                     'currency' => 'gbp',
+                    'metadata' => [
+                        'campaignId' => $donation->getCampaign()->getSalesforceId(),
+                        'charityName' => $donation->getCampaign()->getCharity()->getName(),
+                        'env' => getenv('APP_ENV'),
+                        'isGiftAid' => $donation->isGiftAid(),
```

Comment:
It's probably good to have `coreDonationGiftAid` and `tbgTipGiftAid` and set them both to the main flag's value initially, as it sounds like we plan to split the declaration soon.